File: python/01-iniciante/problem1020.py
# Inputs

days = int(input()) # Days

# Calculations

yea = int(days/(365))  # Years
rest = int(days % 365)
mon = int(rest/30)     # Months
day = int(rest % 30)   # Days

# Outputs

print(str(yea) + " ano(s)")
print(str(mon) + " mes(es)")
print(str(day) + " dia(s)")

File: go/01-iniciante/problem1020.go
package main

import (
    "fmt"
)

func main() {
    var t int   // Time in days

    fmt.Scanf("%d", &t)

    y := t/365   // Years
    r := t % 365 // Rest in days
    m := r/30    // Months
    d := r % 30   // Days

    fmt.Print(y, " ano(s)\n", m, " mes(es)\n", d, " dia(s)")
}
